include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_histogram.h

```diff
@@ -517,19 +518,18 @@ __parallel_histogram_select_kernel(oneapi::dpl::__internal::__device_backend_tag
     // if bins fit into registers, use register private accumulation
     if (__num_bins <= __max_work_item_private_bins)
     {
-        return __future(
-            __histogram_general_registers_local_reduction<__iters_per_work_item, __max_work_item_private_bins>(
-                __backend_tag, ::std::forward<_ExecutionPolicy>(__exec), __init_event, __work_group_size,
-                ::std::forward<_Range1>(__input), ::std::forward<_Range2>(__bins), __binhash_manager));
+        return __histogram_general_registers_local_reduction<__iters_per_work_item, __max_work_item_private_bins>(
```

Comment:
why remove `__future(` here (and below)?

Reply:
Because `__future<sycl::event>` successfully constructable from `sycl::event` which returns from  `__histogram_general_registers_local_reduction`. Should we really want to see additional letters here?
I am ready to add...

Reply:
Restored.